packages/venia-concept/src/components/Currency/index.js

```diff
@@ -0,0 +1,38 @@
+import { Component, createElement } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Currency extends Component {
+
+    static propTypes = {
+        locale: PropTypes.string,
+        value: PropTypes.number.isRequired,
+        currency: PropTypes.string.isRequired,
+        tagName: PropTypes.string
+    };
+
+    static defaultProps = {
+        tagName: 'span'
+    };
+
+    guessLocalLanguage() {
```

Comment:
We don't need to guess here. The spec advises implementers to use the [environment's default](https://www.ecma-international.org/ecma-402/2.0/#sec-defaultlocale). We should trust the browser to do the right thing, unless we have evidence they don't.

Default is accomplished by passing in `undefined` for the first positional argument to `Intl.NumberFormat`. See implementation [here](https://github.com/magento-research/pwa-studio/blob/8539d8752fbe2f24efec41d882efed82d06df959/packages/peregrine/src/Price/Price.js#L23)

Reply:
Good, that's good news--though I'm removing this implementation anyway.